src/Engine/ProtoCore/Lang/BuiltInFunctionEndPoint.cs

```diff
@@ -524,17 +524,6 @@ public override StackValue Execute(ProtoCore.Runtime.Context c, List<StackValue>
                         interpreter, 
                         stackFrame);
                     break;
-                case BuiltInMethods.MethodID.TryGetValueFromNestedDictionaries:
-                    ret = StackValue.Null;
-
-                    if (formalParameters[0].IsArray)
-                    {
-                        StackValue value;
-                        var parameterArray = runtimeCore.Heap.ToHeapObject<DSArray>(formalParameters[0]);
-                        if (parameterArray.TryGetValueFromNestedDictionaries(formalParameters[1], out value, runtimeCore))
-                            ret = value;
-                    }
-                    break;
```

Comment:
This function is used in `FunctionCallNodeController` alone and is removed. There is no way, AFAIK, to call it from a code block even using a `__` prefix.